syncto/__init__.py

```diff
@@ -32,6 +35,9 @@ def main(global_config, **settings):
                        default_settings=DEFAULT_SETTINGS)
 
     settings = config.get_settings()
+    config.registry.heartbeats['sync'] = get_token_server_ping(
+        settings['token_server_url'],
+        settings['token_server_heartbeat_timeout_seconds'])
```

Comment:
I don't see why it helps unit testing actually. Plus, since we are used to modify settings on the fly everywhere in tests, this would make it inconsistent with the others no ?